src/utils/MainApi.js

```diff
@@ -0,0 +1,82 @@
+const baseUrl =
+  'https://api.news-explorer-hartsek.students.nomoredomainssbs.ru';
+const headers = {
+  'content-type': 'application/json',
+};
+
+const checkResponse = (res) => {
```

Comment:
You have 2 functions `checkResponse` in 2 files. Please. define only one for the whole project


so, in `NewsApi.js` it's not needed


export it from here